controllers/apps/apimanager_controller.go

```diff
@@ -143,11 +141,28 @@ func (r *APIManagerReconciler) SetupWithManager(mgr ctrl.Manager) error {
 		Logger:    r.Logger().WithName("APIManagerRoutesHandler"),
 	}
 
+	cmEventMapper := &APIManagerConfigMapsEventMapper{
+		Context:   r.Context(),
+		K8sClient: r.Client(),
+		Logger:    r.Logger().WithName("APIManagerConfigMapsEventMapper"),
+	}
+
 	labelSelectorPredicate, err := predicate.LabelSelectorPredicate(r.SecretLabelSelector)
 	if err != nil {
 		return nil
 	}
 
+	redisConfigLabelSelector := &apimachinerymetav1.LabelSelector{
+		MatchLabels: map[string]string{
+			"threescale_component":         "system",
```

Comment:
we are not concerned about backend-redis?

Reply:
@MStokluska thanks for comment. We are using this label selectot to create redisConfigLabelPredicate, to watch Config map `redis-config`. These are labels that `redis-config` CM has:

```
 $ oc describe cm redis-config
Name:         redis-config
Namespace:    3scale-test
Labels:       app=3scale-api-management
              threescale_component=system
              threescale_component_element=redis
Annotations:  <none>
```
Backend-redis we checked;  logs are present in Validation, and also added below:
```
kubectl exec -ti $(kubectl get pods -l deployment=backend-redis -o name) -- /bin/bash
bash-4.2$ redis-cli 
127.0.0.1:6379> replicaof 10.10.10.10 6379
(error) ERR unknown command `replicaof`, with args beginning with: `10.10.10.10`, `6379`,
127.0.0.1:6379> replicaof
(error) ERR unknown command `replicaof`, with args beginning with: 
127.0.0.1:6379> slaveof
(error) ERR unknown command `slaveof`, with args beginning with: 
127.0.0.1:6379> 
```
backend-redis is working as expected.
Thank you